### PR TITLE
go fmt outputs empty line, sometimes.

### PR DESCRIPTION
**Pull request by merlin-northern**
example job:
https://gitlab.com/Northern.tech/Mender/go-lib-micro/-/jobs/971381446

from this pr:
https://github.com/mendersoftware/go-lib-micro/pull/100

and indeed it happens:
```
#echo "$(go fmt ./...)"

```

ChangeLog:none
Signed-off-by: Peter Grzybowski <peter@northern.tech>

### CODE REVIEW

File: .gitlab-ci-check-golang-static.yml [hunk 1]
```diff
@@ -40,7 +40,7 @@ test:static:
     # Test if code was formatted with 'go fmt'
     # Command will format code and return modified files
     # fail if any have been modified.
-    - if [ -n "$(go fmt ./...)" ]; then echo 'Code is not formatted with "go fmt"'; false; fi
+    - if [ -n "$(go fmt ./... | grep -v ^$)" ]; then echo 'Code is not formatted with "go fmt"'; false; fi
```
Comment (kacf) [inline .gitlab-ci-check-golang-static.yml:43]: I think you need to single-quote `^$`. I'm not sure when the `$` is at the end like that, but I think it's a safer approach anyway.

**Review by lluiscampos — COMMENTED** [review]
I am not sure I understand the consequences of this.

I think this will make the test ignore new lines, but then a local run of `go fmt ./...` could also add these lines back, right? The test will pass the next round also, but why don't be strict which these empty lines in the first place?

Also, if we merge this, @merlin-northern please follow up on pipelines for `mender`, `mender-artifact`, `mender-connect` (possibly others?).

**Comment by alfrunes** [comment]
@merlin-northern ping

**Comment by merlin-northern** [comment]
> @merlin-northern ping

wow. I am putting it on my todolist. I will come back to it.

**Comment by merlin-northern** [comment]
> @merlin-northern ping

wow. I am putting it on my todolist. I will come back to it.

**Comment by lluiscampos** [comment]
Doing the job of `stalebot`... This PR is now too old. Closing.
